scripts/tests.e2e.persistent.sh

```diff
@@ -55,6 +56,10 @@ else
 fi
 
 print_separator
-# Setting E2E_USE_PERSISTENT_NETWORK configures tests.e2e.sh to use
-# the persistent network identified by TESTNETCTL_NETWORK_DIR.
-E2E_USE_PERSISTENT_NETWORK=1 ./scripts/tests.e2e.sh
+# - Setting E2E_USE_PERSISTENT_NETWORK configures tests.e2e.sh to use
+#   the persistent network identified by TESTNETCTL_NETWORK_DIR.
+# - Only a single test is required to validate that a persistent
+#   network can be used by an e2e test suite run. Executing more tests
+#   would be duplicative of the testing performed against an ephemeral
+#   test network.
+E2E_USE_PERSISTENT_NETWORK=1 ./scripts/tests.e2e.sh --ginkgo.focus-file=permissionless_subnets.go
```

Comment:
I'm trying to understand what the rationale for `--ginkgo.focus-file=permissionless_subnets.go` is. My understanding is that we want to just call the `tests.e2e.sh` script enabled on a persistent network, so I don't understand why we're only running this w/ a single test file enabled. Am I missing something here..?